spec/ransack/helpers/form_helper_spec.rb

```diff
@@ -399,6 +399,34 @@ module Helpers
         it { should match /example_name_eq/ }
       end
 
+      describe '#search_form_for with config set to remove sort order indicators' do
+        before do
+          Ransack.configure { |c| c.hide_sort_order_indicators = true }
+        end
+        subject { @controller.view_context
+          .sort_link(
+            [:main_app, Person.search(sorts: ['name desc'])],
+            :name,
+            controller: 'people'
+          )
+        }
+        it { should match /Full Name/ }
```

Comment:
OK for leaving the tests with RSpec `should` syntax since the rest of that file is that way. It might be better to move these 2 tests at the end of the `sort_link` tests where the other hide indicator specs are, and describe them like the other tests too, i.e. 'sort_link with...' instead of `search_form_for with...` since they relate to the sort_link method.